### PR TITLE
Add script for copying resource

### PR DESCRIPTION
**Pull request by Dysproz**
This is helper script which allow to copy resources between two clusters.
I've developed it as a help tool for side-by-side deployment of Openshift (where user has to manually copy CA secret abd csr signer configMap)

### CODE REVIEW

**Comment by Dysproz** [comment]
recheck ha

**Review by psykulsk — APPROVED** [review]

**Review by kaweue — APPROVED** [review]

File: deploy/openshift/tools/resource-copier/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## What do I need to use it?
+CLuster selection is based on kubeconfig file. So in order to access clusters, prepare kubeconfig file for both cluster from which you want to copy resource and cluster where you want to create copy of resource.
+
+## Usage
+"""
+./resource-copier.sh --from-cluster <kubeconfig file of cluster with existing resource> --to-cluster <kubeconfig file of cluster where to copy resource> --from-namespace <target namespace where is original resource> --to-namespace <target namespace where to copy resource> --resource <resource type> --name <name of the resource>
+"""
+
+If one of kubeconfigs is not passed then script will automatically perform copy in scope of the same cluster.
```
Comment (katrybacka) [inline deploy/openshift/tools/resource-copier/README.md:14]: what about a situation when you have multiple clusters defined in different contexts in one kubeconfig file?

**Review by katrybacka — CHANGES_REQUESTED** [review]

File: deploy/openshift/tools/resource-copier/README.md (hunk 1 above)
Comment (katrybacka) [reply deploy/openshift/tools/resource-copier/README.md:14]: if it is not completely a case @Dysproz i will drop this comment but for me it really common to use one kubeconfig for multiple contexts

Comment (Dysproz) [reply deploy/openshift/tools/resource-copier/README.md:14]: Not really in this case. If we'll need that feature then I'll add it for sure :)

**Review by katrybacka — APPROVED** [review]
ok understood

**Pull request merged**